src/Blueprint/Blueprint/help/Import-AzBlueprintWithArtifact.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Blueprint.dll-Help.xml
+Module Name: Az.Blueprint
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blueprint/import-azblueprintwithartifact
+schema: 2.0.0
+---
+
+# Import-AzBlueprintWithArtifact
+
+## SYNOPSIS
+Import a blueprint file in JSON format to a blueprint object and save it within the specified subscription or management group.
```

Comment:
"Import a blueprint definition file..." and "... to a blueprint definition object..."